day_3/ascii_art.py

```diff
@@ -0,0 +1,26 @@
+START_AND_STOP = '_  __  __  __  __  __  __  __  __  __  __  __  __  __  __  __  __  __  __  __  _\n' \
+                 ' )(  )(  )(  )(  )(  )(  )(  )(  )(  )(  )(  )(  )(  )(  )(  )(  )(  )(  )(  )( \n' \
+                 '(__)(__)(__)(__)(__)(__)(__)(__)(__)(__)(__)(__)(__)(__)(__)(__)(__)(__)(__)(__)'
+TOP_BORDER = '<+><+><+><+><+><+><+><+><+><+><+><+><+><+><+><+><+><+><+><+><+><+><+><+><+><+><+>'
```

Comment:
_nit_: since these have fixed length, I'd suggest making sure that every message on the screen will be no longer than this fixed length, just for better appearance.